core/src/Rpc/Client.cs

```diff
@@ -736,19 +736,6 @@ public Task<IList<Device>> GetPairedDevices()
                 .ToTask(response => (IList<Device>)response.Devices);
         }
 
```

Comment:
I thought we were not using core project at all. Is that not correct? If yes, then why change anything?

Reply:
Why was this added and why is this deleted now?

Reply:
> I thought we were not using core project at all. Is that not correct? If yes, then why change anything?

Core project is base project for Tpp and User projects. Tpp and User use Core as project dependency.

Reply:
> Why was this added and why is this deleted now?

With latest version of Gateway.proto, I don't see any message field for VerifyAffiliateRequest and VerifyAffiliateResponse. Neither this method is present in SDK java as well, so it was removed